Bottom: 1495dcd5efdf5475ac49a80f4a018974eb414f79
Top:    ce668d8e5a4e0bb3756869ff3f3996f1bc43fb7c
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-10-09 11:21:58 -0700

Refresh of 5-5-librdmacm-rping-added-chec

---

diff --git a/examples/rping.c b/examples/rping.c
index d702592..1c4688c 100644
--- a/examples/rping.c
+++ b/examples/rping.c
@@ -1221,11 +1221,12 @@ int main(int argc, char *argv[])
 			ret = rping_run_persistent_server(cb);
 		else
 			ret = rping_run_server(cb);
-	} else
+	} else {
 		ret = rping_run_client(cb);
+	}
 
 	DEBUG_LOG("destroy cm_id %p\n", cb->cm_id);
-	ret = rdma_destroy_id(cb->cm_id);
+	rdma_destroy_id(cb->cm_id);
 out2:
 	rdma_destroy_event_channel(cb->cm_channel);
 out:
